packages/cli/lib/lib/webpack/render-html-plugin.js

```diff
@@ -50,8 +50,8 @@ module.exports = function(config) {
 			config,
 			url,
 			ssr(params) {
-				Object.assign(props, params, { url });
-				return config.prerender ? prerender({ cwd, dest, src }, props) : '';
+				Object.assign(params, { url });
```

Comment:
well now this line is kinda pointless 👀

Reply:
The only reason I didn't remove was I just wanted to know if that was your intent.

I just didn't know if you wanted to remove the ability to pass params to ssr entirely or just extras beyond url.

Reply:
yeah just remove it

Reply:
Alright i've made those changes. I also removed the option that gets passed in the template, let me know if you want me to bring that back. Just didn't want to leave dead code there.